src/data/tags.ts

```diff
@@ -70,6 +77,13 @@ export const tags: Tag[] = [
     title: "How does a PR/MR count for Hacktoberfest?",
     aliases: ["pr", "mrs", "mr"],
   },
+  {
+    name: "pr-count",
+    content:
+      "You need four PRs/MRs accepted between October 1 and October 31 to complete Hacktoberfest",
+    title: "How many PRs/MRs do I need?",
+    aliases: ["mr-count"],
```

Comment:
```suggestion
    aliases: ["mr-count", "count"],
```
Might be good to alias `count` too.